src/test/java/apoc/load/relative/LoadXmlRelativePathTest.java

```diff
@@ -0,0 +1,268 @@
+                            "| \"bk103\" | \"Corets, Eva\"          | \"Maeve Ascendant\"                        |\n" +
+                            "| \"bk104\" | \"Corets, Eva\"          | \"Oberon's Legacy\"                        |\n" +
+                            "| \"bk105\" | \"Corets, Eva\"          | \"The Sundered Grail\"                     |\n" +
+                            "| \"bk106\" | \"Randall, Cynthia\"     | \"Lover Birds\"                            |\n" +
+                            "| \"bk107\" | \"Thurman, Paula\"       | \"Splish Splash\"                          |\n" +
+                            "| \"bk108\" | \"Knorr, Stefan\"        | \"Creepy Crawlies\"                        |\n" +
+                            "| \"bk109\" | \"Kress, Peter\"         | \"Paradox Lost\"                           |\n" +
+                            "| \"bk110\" | \"O'Brien, Tim\"         | \"Microsoft .NET: The Programming Bible\"  |\n" +
+                            "| \"bk111\" | \"O'Brien, Tim\"         | \"MSXML3: A Comprehensive Guide\"          |\n" +
+                            "| \"bk112\" | \"Galos, Mike\"          | \"Visual Studio 7: A Comprehensive Guide\" |\n" +
+                            "+-----------------------------------------------------------------------------+\n" +
+                            "12 rows\n", result.resultAsString());
+                });
+    }
+
+    @Test
+    public void testLoadXmlXpathAuthorFromBookId () {
+        testCall(db, "CALL apoc.load.xml('books.xml', '/catalog/book[@id=\"bk102\"]/author') yield value as result",
+                (r) -> {
+                    assertEquals("author", ((Map) r.get("result")).get("_type"));
+                    assertEquals("Ralls, Kim", ((Map) r.get("result")).get("_text"));
+                });
+    }
+
+    @Test
+    public void testLoadXmlXpathGenreFromBookTitle () {
+        testCall(db, "CALL apoc.load.xml('books.xml', '/catalog/book[title=\"Maeve Ascendant\"]/genre') yield value as result",
+                (r) -> {
+                    assertEquals("genre", ((Map) r.get("result")).get("_type"));
+                    assertEquals("Fantasy", ((Map) r.get("result")).get("_text"));
+                });
+    }
+
+    @Test
+    public void testLoadXmlXpathReturnBookFromBookTitle () {
+        testCall(db, "CALL apoc.load.xml('books.xml', '/catalog/book[title=\"Maeve Ascendant\"]/.') yield value as result",
+                (r) -> {
+                    Object value = r.values();
+                    assertEquals(XML_XPATH_AS_NESTED_MAP, value.toString());
+                });
+    }
+
+    @Test
+    public void testLoadXmlXpathBooKsFromGenre () {
+            testResult(db, "CALL apoc.load.xml('books.xml', '/catalog/book[genre=\"Computer\"]') yield value as result",
+                (r) -> {
+                    Map<String, Object> next = r.next();
+                    Object result = next.get("result");
```

Comment:
```suggestion
                    Map<String, Object> result = (Map<String, Object>) next.get("result");
```
e aggiustare di conseguenza

Reply:
**Da fare**